Gemfile

```diff
@@ -2,6 +2,9 @@ source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
 gemspec
 
+gem 'activesupport', '~>5.2.3'
```

Comment:
we shouldn't depend on activesupport, or at least, I'm very suspicious of this. let me take a look

Reply:
Locally, with an updated bundle, activesupport and rubyzip are already pulled in as dependencies of markdown, which is marked as a dependency in the gemspec (most likely for historical reasons that may or may not be valid, but let's ignore that for the moment).

It wasn't necessary to make an explicit dependency between beaker-answers and these gems locally though; are you seeing something different?

Reply:
Here is the failed job https://jenkins-sre.delivery.puppetlabs.net/job/qe_beaker-answers_unit-ruby_ghpr_master/18/.  My local Gemfile.lock has set activesupport to 5.2.3 and it passes spec

Reply:
Ok, the errors in the job are due to ruby version constraints on zeitworks and activesupport, both of which are present as dependencies that tie back to markdown, which is marked as a dependency in the gemspec.  A current bundle needs Ruby 2.5.0, or we can get rid of markdown, or pin that if need be.

Reply:
So let's get rid of the markdown dependency; I don't see anything in the gem that uses it. Do you?

Reply:
I ran yard, which took for bloody ever for reasons that are passing mysterious since there's next to nothing in this gem. I'm not sure what else would need markdown, and if something did, it should be declared in its dependencies... :man_shrugging:

Reply:
I was thinking markdown was needed for yard docs. Looks like it is not. Removed markdown and ran yard successfully locally

Reply:
Tests passed https://jenkins-sre.delivery.puppetlabs.net/job/qe_beaker-answers_unit-ruby_ghpr_master/22/

Reply:
:+1: